src/js/modules/infragistics.ui.videoplayer.js

```diff
@@ -49,223 +49,1826 @@
+				/* type="bool" Get or set whether to apply animation effects when showing or hiding the banner. If set to true, the animation is played for banner.duration in milliseconds.
+				```
+					//Initialize
+					$(".selector").igVideoPlayer({
+							banners: [{
+								imageUrl: "../Content/images/Banner1.png",
+								link: "http://quince.infragistics.com/",
+								times: [5, 20, 60],
+								visible: false,
+								closeBanner: true,
+								animate: true,
+								autohide: true,
+								hidedelay: 10000,
+								width: "200px",
+								height: "67px"
+							}]
+						});
+
+					//Get
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					var animate = banners[0].animate;
+
+					//Set
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					banners[0].animate = true;
+				```
+				*/
 				animate: true,
-				/* type="bool" Get or set whether the banner is visible or not. */
+				/* type="bool" Get or set whether the banner is visible or not.
+				```
+					//Initialize
+					$(".selector").igVideoPlayer({
+							banners: [{
+								imageUrl: "../Content/images/Banner1.png",
+								link: "http://quince.infragistics.com/",
+								times: [5, 20, 60],
+								visible: false,
+								closeBanner: true,
+								animate: true,
+								autohide: true,
+								hidedelay: 10000,
+								width: "200px",
+								height: "67px"
+							}]
+						});
+
+					//Get
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					var visible = banners[0].visible;
+
+					//Set
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					banners[0].visible = false;
+				```
+				*/
 				visible: true,
-				/* type="number" Get or set the banner animation duration. */
+				/* type="number" Get or set the banner animation duration.
+				```
+					//Initialize
+					$(".selector").igVideoPlayer({
+							banners: [{
+								imageUrl: "../Content/images/Banner1.png",
+								link: "http://quince.infragistics.com/",
+								times: [5, 20, 60],
+								visible: false,
+								closeBanner: true,
+								animate: true,
+								autohide: true,
+								duration: 2000,
+								hidedelay: 10000,
+								width: "200px",
+								height: "67px"
+							}]
+						});
+
+					//Get
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					var duration = banners[0].duration;
+
+					//Set
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					banners[0].duration = 2000;
+				```
+				*/
 				duration: 1000,
-				/* type="bool" Get or set whether to automatically hide the banner. If set to true, the banner is hidden after banner.hidedelay in milliseconds. */
+				/* type="bool" Get or set whether to automatically hide the banner. If set to true, the banner is hidden after banner.hidedelay in milliseconds.
```

Comment:
hidedelay should be an option link:

`[hidedelay](ui.igvideoplayer#options:banners.hidedelay)`